app/components/Project/ProjectMilestoneDue.tsx

```diff
@@ -15,15 +15,18 @@ const ProjectMilestoneDue: React.FC<Props> = ({ project }) => {
     graphql`
       fragment ProjectMilestoneDue_project on Project {
         latestCommittedProjectRevision {
-          upcomingReportingRequirementFormChange {
+          upcomingReportingRequirementFormChange(reportType: "Milestone") {
             asReportingRequirement {
               reportDueDate
             }
           }
         }
         latestCompletedReportingRequirements: reportingRequirementsByProjectId(
           orderBy: REPORT_DUE_DATE_ASC
-          filter: { submittedDate: { isNull: false } }
+          filter: {
+            submittedDate: { isNull: false }
+            reportType: { includes: "Milestone" }
```

Comment:
there is an `isMilestone` column that we should be able to use in the `ReportType`, if we want to be more resilient to changes

Reply:
![image](https://user-images.githubusercontent.com/2684022/177864583-5c19bb64-515e-4928-aa6c-31060baff3de.png)

@BCerki  the `isMilestone` field is in `reportType`, so you have to follow the `reportTypeByReportType` foreign key